api/src/models/Task.ts

```diff
@@ -1,83 +1,18 @@
-import { DataTypes, Model, Sequelize, Deferrable } from 'sequelize';
+// import sequelize which we create in db file
+  const sequelize = require('../orm');
 
-class Task extends Model {}
+// из пакета sequelize нам потребуется импортировать класс с помощью которого описываются типы того или иного поля
+const { DataTypes } = require('sequelize');
 
-const initialTaskModel = (sequelize: Sequelize) => {
-  Task.init(
```

Comment:
I'm sorry for not clarifying sooner that we already have the Task model in place.🙇